PocketCastsTests/Tests/IAP/IAPHelperTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+import StoreKitTest
+
+@testable import podcasts
+import PocketCastsServer
+
+final class IAPHelperTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    let configurationFile = "Pocket Casts Configuration"
+
+    func testRequestInfo() throws {
+        let session = try SKTestSession(configurationFileNamed: configurationFile)
+        session.clearTransactions()
+        session.resetToDefaultState()
+        session.disableDialogs = true
+
+        let helper = IAPHelper()
+        let expectation = XCTestExpectation(description: "Fetch Products")
+        NotificationCenter.default.addObserver(forName: ServerNotifications.iapProductsUpdated, object: nil, queue: nil) { notification in
+            expectation.fulfill()
+        }
+        helper.requestProductInfo()
+
+        wait(for: [expectation], timeout: 5)
+        XCTAssert(helper.hasLoadedProducts)
+
+        let _ = helper.getProduct(for: .monthly)
```

Comment:
Do we need this line? We're already calling `requestProductInfo()` directly

Reply:
We are just checking if we get a product without an error after doing a request. If you notice the signature method of getProductInfo has a ! on the return type. I'm not happy with that and I'm considering to change it for a ? and check if a get a non nil.